web/frontend/src/lib/components/robot-client.svelte

```diff
@@ -495,17 +502,33 @@ if (supportedAuthTypes.length === 0) {
     <div class="px-4 py-3">
       <span>{authType}: </span>
       <div class="w-96">
-        <input
-          bind:value={password}
-          disabled={$connectionStatus === 'connecting'}
-          class="
+        {#if authType === "api-key"}
+          <label class="text-sm">
+            auth entity
+            <input
+                    bind:value={authEntityByAuthType[authType]}
+                    disabled={$connectionStatus === 'connecting'}
+                    class="
+              mb-2 block w-full appearance-none border p-2 text-gray-700
+              transition-colors duration-150 ease-in-out placeholder:text-gray-400 focus:outline-none
+            "
+                    autocomplete="off"
+            ></label>
+        {/if}
+        <label class="text-sm">
+          payload
+          <input
+                  bind:value={passwordByAuthType[authType]}
+                  disabled={$connectionStatus === 'connecting'}
+                  class="
             mb-2 block w-full appearance-none border p-2 text-gray-700
             transition-colors duration-150 ease-in-out placeholder:text-gray-400 focus:outline-none
           "
-          type="password"
-          autocomplete="off"
-          on:keyup={async (event) => event.key === 'Enter' && login(authType)}
-        >
+                  type="password"
+                  autocomplete="off"
+                  on:keyup={async (event) => event.key === 'Enter' && login(authType)}
+          >
+        </label>
         <v-button
           disabled={$connectionStatus === 'connecting'}
           label="Login"
```

Comment:
GitHub won't let me put this comment on the `on:click` line for the `v-button` 🤦🏻‍♂️

I would forgo the conditional for "connecting" since the `v-button` should prevent interaction while disabled. Maybe that's why the login button isn't working?

```
on:click={async () => login(authType)}
```

Reply:
weirdly this issue went away when i converted from `v-button` -> `button` as part of my minor makeover here.